crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -0,0 +1,377 @@
+use masking::PeekInterface;
+use serde::{Deserialize, Serialize};
+use time::PrimitiveDateTime;
+use url::Url;
+use uuid::Uuid;
+
+use crate::{
+    core::errors,
+    pii::{self, Secret},
+    services,
+    types::{self, api, storage::enums},
+};
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexIntentRequest {
+    request_id: String,
+    amount: i64,
+    currency: enums::Currency,
+    merchant_order_id: String,
+}
+impl TryFrom<&types::PaymentsAuthorizeSessionTokenRouterData> for AirwallexIntentRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: &types::PaymentsAuthorizeSessionTokenRouterData,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            request_id: Uuid::new_v4().to_string(),
```

Comment:
What this `request_id` refers to?

Reply:
Unique ID to be sent for each transaction/operation request to the connector